data/remnant/remnant 2 side missions.txt

```diff
@@ -400,6 +401,9 @@ mission "Remnant: Teciimach Deployment"
 	on complete
 		dialog
 			`Now that the most recent Korath raiders have been defeated, you decide to check your sensor logs to see how the new EMP torpedoes Taely talked about performed. You note that the EMP bursts no longer have an impact on friendly ships. This could make them much more versatile.`
+	on fail
+		dialog
+			`You recall Taely mentioning something about new weaponry, so you decide to check your sensor logs to see what your sensors have recorded lately. Looking over the most recent combat logs, you note that the EMP bursts no longer have an impact on friendly ships. This could make them much more versatile.`
```

Comment:
Actually, too many `sensors`.
```suggestion
			`You recall Taely mentioning something about new weaponry, so you decide to check your logs to see what your sensors have recorded lately. Looking over the most recent combat logs, you note that the EMP bursts no longer have an impact on friendly ships. This could make them much more versatile.`
```
```suggestion
			`You recall Taely mentioning something about new weaponry, so you decide to check your sensor logs to see what they have recorded lately. Looking over the most recent combat logs, you note that the EMP bursts no longer have an impact on friendly ships. This could make them much more versatile.`
```